### PR TITLE
Update lambda ARNs for release v0.26.0

### PR DESCRIPTION
**Pull request by Kausik-A**
- Update lambda docs with new ARNs. 
- Add blog post for v0.26.0 release

### CODE REVIEW

File: src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-lambda-layer-v0.26.0.mdx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+|US East (N. Virginia)    |  ✓    |  ✓    |
+|US East (Ohio)           |  ✓    |  ✓    |
+|US West (N California)   |  ✓    |  ✓    |
+|US West (Oregon)         |  ✓    |  ✓    |
+
+<SectionSeparator />
+
+### Release Highlights
+
+- Python layer [**aws-otel-python-<amd64|arm64>-ver-1-15-0**](https://aws-otel.github.io/docs/getting-started/lambda/lambda-python) contains OpenTelemetry Python `v1.16.0`
```
Comment (vasireddy99) [inline src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-lambda-layer-v0.26.0.mdx:36]:
```suggestion
- Python layer [**aws-otel-python-<amd64|arm64>-ver-1-16-0**](https://aws-otel.github.io/docs/getting-started/lambda/lambda-python) contains OpenTelemetry Python `v1.16.0`
```

File: src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-lambda-layer-v0.26.0.mdx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+- Nodejs layer [**aws-otel-nodejs-<amd64|arm64>-ver-1-8-0**](https://aws-otel.github.io/docs/getting-started/lambda/lambda-js) contains OpenTelemetry JavaScript Core `v1.9.1` with AWS Lambda Instrumentation `v0.35.0`
```
Comment (vasireddy99) [inline src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-lambda-layer-v0.26.0.mdx:37]:
```suggestion
- Nodejs layer [**aws-otel-nodejs-<amd64|arm64>-ver-1-9-1**](https://aws-otel.github.io/docs/getting-started/lambda/lambda-js) contains OpenTelemetry JavaScript Core `v1.9.1` with AWS Lambda Instrumentation `v0.35.0`
```

File: src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-lambda-layer-v0.26.0.mdx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+- Java-Wrapper layer [**aws-otel-java-wrapper-<amd64|arm64>-ver-1-21-0**](https://aws-otel.github.io/docs/getting-started/lambda/lambda-java) contains OpenTelemetry Java `v1.21.1`
+- Java-Agent layer [**aws-otel-java-agent-<amd64|arm64>-ver-1-21-0**](https://aws-otel.github.io/docs/getting-started/lambda/lambda-java-auto-instr) contains AWS Distro for OpenTelemetry Java Instrumentation `v1.21.1`
```
Comment (vasireddy99) [inline src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-lambda-layer-v0.26.0.mdx:39]:
```suggestion
- Java-Agent layer [**aws-otel-java-agent-<amd64|arm64>-ver-1-21-1**](https://aws-otel.github.io/docs/getting-started/lambda/lambda-java-auto-instr) contains AWS Distro for OpenTelemetry Java Instrumentation `v1.21.1`
```

File: src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-lambda-layer-v0.26.0.mdx [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+|US West (N California)   |  ✓    |  ✓    |
+|US West (Oregon)         |  ✓    |  ✓    |
+
+<SectionSeparator />
+
+### Release Highlights
+
+- Python layer [**aws-otel-python-<amd64|arm64>-ver-1-15-0**](https://aws-otel.github.io/docs/getting-started/lambda/lambda-python) contains OpenTelemetry Python `v1.16.0`
+- Nodejs layer [**aws-otel-nodejs-<amd64|arm64>-ver-1-8-0**](https://aws-otel.github.io/docs/getting-started/lambda/lambda-js) contains OpenTelemetry JavaScript Core `v1.9.1` with AWS Lambda Instrumentation `v0.35.0`
+- Java-Wrapper layer [**aws-otel-java-wrapper-<amd64|arm64>-ver-1-21-0**](https://aws-otel.github.io/docs/getting-started/lambda/lambda-java) contains OpenTelemetry Java `v1.21.1`
```
Comment (vasireddy99) [inline src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-lambda-layer-v0.26.0.mdx:38]:
```suggestion
- Java-Wrapper layer [**aws-otel-java-wrapper-<amd64|arm64>-ver-1-21-1**](https://aws-otel.github.io/docs/getting-started/lambda/lambda-java) contains OpenTelemetry Java `v1.21.1`
```

File: src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-lambda-layer-v0.26.0.mdx [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+- Java-Agent layer [**aws-otel-java-agent-<amd64|arm64>-ver-1-21-0**](https://aws-otel.github.io/docs/getting-started/lambda/lambda-java-auto-instr) contains AWS Distro for OpenTelemetry Java Instrumentation `v1.21.1`
+- Collector layer **aws-otel-collector-<amd64|arm64>-ver-0-68-0** contains ADOT Collector for Lambda `v0.26.0`. Compatible with [.NET](https://aws-otel.github.io/docs/getting-started/lambda/lambda-dotnet) and [Go](https://aws-otel.github.io/docs/getting-started/lambda/lambda-go) runtimes.
```
Comment (vasireddy99) [inline src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-lambda-layer-v0.26.0.mdx:40]:
```suggestion
- Collector layer **aws-otel-collector-<amd64|arm64>-ver-0-70-0** contains ADOT Collector for Lambda `v0.26.0`. Compatible with [.NET](https://aws-otel.github.io/docs/getting-started/lambda/lambda-dotnet) and [Go](https://aws-otel.github.io/docs/getting-started/lambda/lambda-go) runtimes.
```

File: src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-lambda-layer-v0.26.0.mdx [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+We also welcome you to participate in the [OpenTelemetry project](https://github.com/open-telemetry). The project was [approved for incubation](https://www.cncf.io/blog/2021/08/26/opentelemetry-becomes-a-cncf-incubating-project/) status in August 2021 by the Cloud Native Computing Foundation Technical Oversight Committee (CNCF TOC).
+
+More blog posts about
+[AWS Distro for OpenTelemetry](https://aws.amazon.com/blogs/opensource/category/management-tools/aws-distro-for-opentelemetry/) can be found on the
+[AWS Open Source Blog](https://aws.amazon.com/blogs/opensource/category/management-tools/aws-distro-for-opentelemetry/).
+
+
+### Author
+
+[Anthony Mirabella](https://github.com/Aneurysm9) is a Senior SDE on the AWS Open-Source Observability team.  He is a maintainer of the OpenTelemetry Go client library and Lambda extensions.
```
Comment (vasireddy99) [inline src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-lambda-layer-v0.26.0.mdx:55]: Kausik :-)

File: src/docs/getting-started/lambda/lambda-java.mdx [hunk 7]
```diff
@@ -39,7 +39,7 @@ Find the supported regions and amd64(x86_64)/arm64 layer ARN in the table below
...
+| ap-northeast-1<br/>ap-northeast-2<br/>ap-south-1<br/>ap-southeast-1<br/>ap-southeast-2<br/>ca-central-1<br/>eu-central-1<br/>eu-north-1<br/>eu-west-1<br/>eu-west-2<br/>eu-west-3<br/>sa-east-1<br/>us-east-1<br/>us-east-2<br/>us-west-1<br/>us-west-2 | arn:aws:lambda:<region\>:901920570463:layer:aws-otel-java-wrapper-<architecture\>-ver-1-21-1:1 | Contains [OpenTelemetry for Java v1.22.0](https://github.com/open-telemetry/opentelemetry-java/releases/tag/v1.22.0) with [Java Instrumentation v1.22.0](https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/tag/v1.22.0) <br/><br/> Contains the [ADOT Collector for Lambda v0.26.0](https://github.com/aws-observability/aws-otel-collector/releases/tag/pkg%2Flambdacomponents%2Fv0.26.0) |
```
Comment (vasireddy99) [inline src/docs/getting-started/lambda/lambda-java.mdx:42]: oh did we include Otel- Java v0.22.0 ? 

```suggestion
| ap-northeast-1<br/>ap-northeast-2<br/>ap-south-1<br/>ap-southeast-1<br/>ap-southeast-2<br/>ca-central-1<br/>eu-central-1<br/>eu-north-1<br/>eu-west-1<br/>eu-west-2<br/>eu-west-3<br/>sa-east-1<br/>us-east-1<br/>us-east-2<br/>us-west-1<br/>us-west-2 | arn:aws:lambda:<region\>:901920570463:layer:aws-otel-java-wrapper-<architecture\>-ver-1-21-1:1 | Contains [OpenTelemetry for Java v1.22.0](https://github.com/open-telemetry/opentelemetry-java/releases/tag/v1.22.0) with [Java Instrumentation v1.22.0](https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/tag/v1.21.0) <br/><br/> Contains the [ADOT Collector for Lambda v0.26.0](https://github.com/aws-observability/aws-otel-collector/releases/tag/pkg%2Flambdacomponents%2Fv0.26.0) |
```

Comment (Kausik-A) [reply src/docs/getting-started/lambda/lambda-java.mdx:42]: Yeah, it is referenced[ here](https://github.com/aws-observability/aws-otel-lambda/blob/a5b62b89cb31e46da799757c0dca694c6662e322/java/build.gradle.kts#L27)

File: src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-lambda-layer-v0.26.0.mdx (hunk 6 above)
Comment (Kausik-A) [reply src/content/Blogs/ReleaseBlogs/aws-distro-for-opentelemetry-lambda-layer-v0.26.0.mdx]: Oops, fixed

**Review by rapphil — APPROVED** [review]

File: src/docs/getting-started/lambda/lambda-java.mdx (hunk 7 above)
Comment (vasireddy99) [reply src/docs/getting-started/lambda/lambda-java.mdx:42]: We would need to change[ instrumentation tag](https://github.com/aws-observability/aws-otel-lambda/blob/a5b62b89cb31e46da799757c0dca694c6662e322/java/build.sh#L23) to `v0.21.0`

**Pull request merged**